checker/src/main/java/org/checkerframework/checker/resourceleak/ResourceLeakTransfer.java

```diff
@@ -138,6 +143,14 @@ public TransferResult<AccumulationValue, AccumulationStore> visitObjectCreation(
    */
   public void updateStoreWithTempVar(
       TransferResult<AccumulationValue, AccumulationStore> result, Node node) {
+    // If the node is a void method invocation then do not create temp vars for it.
+    if (node instanceof MethodInvocationNode) {
+      MethodInvocationTree methodInvocationTree = (MethodInvocationTree) node.getTree();
+      ExecutableElement executableElement = TreeUtils.elementFromUse(methodInvocationTree);
+      if (ElementUtils.getType(executableElement).getKind() == TypeKind.VOID) {
```

Comment:
Did you test that the type of the element is void in these cases?  I would have thought we would need to write some code to get the return type and check that.

Reply:
I haven't added any new test case to check that. Checked using the existing test cases. I'll add a new test case.

Reply:
Nevermind, I looked at [this similar code](https://github.com/typetools/checker-framework/blob/be2e6d5f436b70df40a3ab51f8b7f61a0e6fc7de/checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallConsistencyAnalyzer.java#L1855-L1860) and it is doing the same check so this is fine.

Adding a test for this directly will be hard since it would depend on the temporary variable names or something like that.  I don't think it's necessary as long as extant tests are passing.